packages/luis-integration/src/types.ts

```diff
@@ -17,10 +17,10 @@ export interface LuisPredictionCallContext {
 interface LuisEntityBase extends Entity {
     readonly $raw: any;
     readonly value: any;
-    readonly utteranceOffsets: {
+    readonly utteranceOffsets?: {
         startIndex: number;
         endIndex: number;
-        length: number;
+        length?: number;
```

Comment:
Related/same as above:

Trying to figure out how this got in here...? I remember doing some work right after M2 to add this and I honestly forgot where it left off, but I'm wondering why it's showing up here in this PR now?